src/selectors/boardSelectors/index.js

```diff
@@ -1 +1,9 @@
-export const selectUserBoards = (state) => state.board.boards;
\ No newline at end of file
+export const selectUserBoards = (state) => state.board.boards;
+
+export const selectBoardUsers = (state) => state.board.boardUsers;
+
+export const selectIsLoading = (state) => state.board.isLoading;
+
+export const selectError = (state) => state.board.error;
+
+export const selectIsError = (state) => state.board.isError;
```

Comment:
I think it is better to have the same naming in selectors, somewhere you have `get...`, somewhere `select...`